libs/blocks/merch/merch.js

```diff
@@ -422,14 +423,7 @@ export async function initService(force = false) {
   }
   const { env, commerce = {}, locale } = getConfig();
   initService.promise = initService.promise ?? polyfills().then(async () => {
-    const { hostname, searchParams } = new URL(window.location.href);
-    let commerceLibPath = '../../deps/mas/commerce.js';
-    if (/hlx\.(page|live)$|localhost$|www\.stage\.adobe\.com$/.test(hostname)) {
-      const maslibs = searchParams.get('maslibs');
-      if (maslibs) {
-        commerceLibPath = `${getMasBase(hostname, maslibs)}/libs/commerce.js`;
-      }
-    }
+    const commerceLibPath = '../../deps/mas/commerce.js';
```

Comment:
I think line and next one is not needed anymore.

Reply:
After remove those, how do you want to define `commerceLib.init()`  from next line?

Reply:
moving it to line 5 should work no?

Reply:
Maybe I am not getting it but, my question is,

The function scope goes like initService() -> initService.promise.then() -> commerceLib.init().
And the commerceLib is defined relying on importing ('../../deps/mas/commerce.js')
I am not sure how we want to define
<img width="694" alt="image" src="https://github.com/user-attachments/assets/9e72381a-c459-4ae3-8330-546361a54d4d">

Should I just put `const commerceLib = await import('../../deps/mas/commerce.js');` on top instead `import '../../deps/mas/commerce.js';` ?

Reply:
`import commerceLib from '../../deps/mas/commerce.js';` should work.

Reply:
I don't think `commerceLib` is available from commerce.js.
I tested and it returns an error
`Failed loading merch SyntaxError: The requested module '../../deps/mas/commerce.js' does not provide an export named 'default' (at merch.js:5:8)`

Instead,
`const commerceLib = await import('../../deps/mas/commerce.js');` on line 6.

Worked fine. (await is still needed).

Reply:
sorry @seanchoi-dev  it should be

`import * as commerceLib from '../../deps/mas/commerce.js';`

I just tried and it works.
I would like to clean up the code and get rid of code that is not needed any more.